spec/controllers/users_controller_spec.rb

```diff
@@ -3,19 +3,26 @@
 require 'rails_helper'
 
 RSpec.describe UsersController, type: :controller do
-  describe 'GET#index' do
+  describe 'GET#new' do
     it 'renders a successful response' do
-      get :index
+      get :new
 
       expect(response).to be_successful
     end
-  end
 
-  describe 'GET#new' do
-    it 'renders a successful response' do
+    it 'renders a correct template' do
       get :new
 
-      expect(response).to be_successful
+      expect(response).to render_template(:new)
+    end
+
+    it 'redirect to home when session exist' do
```

Comment:
redirects

Reply:
Fixed in 7df1200911d25bf2c4c9e17570c62b21ec341c54